### ISSUE TITLE
animate bug

### ISSUE THREAD

**Issue by zoepage** [webcompat/design#44]
<img width="894" alt="screen shot 2017-11-09 at 7 53 47 pm" src="https://user-images.githubusercontent.com/562354/32623860-ce7eccd2-c587-11e7-85ee-8edcc8baffbd.png">
It would be a lovely nice to have to animate the bug to move towards the bulb or something like that (on first load).

We can set a status in the local storage after animating, so this happens only once.
